ionic_user_interface/src/views/UserRoutes.vue

```diff
@@ -0,0 +1,99 @@
+<template>
```

Comment:
This page needs to show something like a "You have not uploaded any routes" when the user has no routes.